.gitignore

```diff
@@ -27,6 +26,10 @@ missing
 # Recommended build directory.
 /build/
 
+# Other deps
+calico-node-*.tar
+docker-compose
+
 # OSX Resource
 .DS_Store
```

Comment:
Again, not necessarily related to this change but this stuff IMO belongs in a user's global configuration.  No reason to have OS-specific targets here.  Mind cleaning this up?